style/MainView.mcss

```diff
@@ -131,6 +131,33 @@
   top: -1px;
 }
 
+.logo-container-outer {
+  stroke: none;
+  stroke-width: 1;
+  fill: none;
+}
+
+.logo-container-inner {
+  stroke-width: 2;
+  stroke: #ffffff;
+}
+
+.networkActive .logo-container-inner {
```

Comment:
This is where I meant to leave that comment.  I didn't use `:global(.logo-container-inner)`, though it seems to be working.